server/start.go

```diff
@@ -127,6 +129,20 @@ which accepts a path for the resulting pprof file.
 				return err
 			}
 
+			home := cast.ToString(serverCtx.Viper.Get(flags.FlagHome))
+			v := viper.New()
+			v.AddConfigPath(filepath.Join(home, "config"))
+			v.SetConfigName("client")
```

Comment:
Is client.toml supposed to inject into server context, and what about the other items in client.toml, what actual issue does this solve?

Reply:
either client.toml or config.toml, but just to allow read config from viper with fallback from genesis like how sdk [handle](https://github.com/cosmos/cosmos-sdk/blob/c0f1f61933191b2d6f747d20f6e70bf6815c92bd/server/util.go#L440)